spec/integration/action_spec.rb

```diff
@@ -428,4 +428,51 @@ def b
       end
     end
   end
+
+  describe 'optional values' do
+    let(:complex_action) do
+      Class.new do
+        include Verbalize::Action
+        input optional: [:no_default, default_const: 3, default_method: ->() { Date.today }]
+
+        def call
+          {
+            no_default:     no_default,
+            default_const:  default_const,
+            default_method: default_method
+          }
+        end
+      end
+    end
+
+    it 'wraps value consts in methods' do
+      expect(complex_action.defaults[:default_const]).to respond_to(:call)
```

Comment:
Like I said above, I don't think we need to expose defaults publicly. This also seems to be testing the internal implementation of Verbalize. This example could just ensure that we get 3 back instead of something else or any error.